pkg/pbq/pbqmanager.go

```diff
@@ -0,0 +1,134 @@
+package pbq
+
+import (
+	"context"
+	"errors"
+	dfv1 "github.com/numaproj/numaflow/pkg/apis/numaflow/v1alpha1"
+	"github.com/numaproj/numaflow/pkg/pbq/store"
+	"github.com/numaproj/numaflow/pkg/pbq/store/memory"
+	"github.com/numaproj/numaflow/pkg/shared/logging"
+	"go.uber.org/zap"
+	"sync"
+)
+
+var NonExistentPBQError error = errors.New("missing PBQ for the partition")
```

Comment:
```suggestion
var NonExistentPBQErr error = errors.New("missing PBQ for the partition")
```

some place we suffix with `Error`, some places we do not. let's always suffix with `Err`